### DISCUSSION TITLE
/mnt/ipedtmp/ grows in size and breaks processing.

### DISCUSSION THREAD

**Discussion by MariasStory** [sepinf-inc/IPED#387] [Q&A]
Hi,
I am using docker image and see that the temp folder is growing in size and breaks processing:
252G    /mnt/ipedtmp/
The processing stops because no space in root directory.
What can I do?

**Comment by lfcnassif** [comment]
Have you enabled indexTempOnSSD in LocalConfig.txt and kept outputOnSSD = false? With that configuration, the whole index is created in indexTemp and copied to output at the end.

Try to disable indexTempOnSSD or to turn outputOnSSD on if that is the case.

**Comment by lfcnassif** [comment]
I saw at https://github.com/iped-docker/iped/issues/5 the recommendation solved your issue. Will move this to discussions, maybe it can help others in the future.
